onnxruntime/core/optimizer/conv_bn_fusion.cc

```diff
@@ -124,24 +113,23 @@ Status ConvBNFusion::Apply(Graph& graph, Node& node, RewriteRuleEffect& rule_eff
   }
 
   // Replace initializers of conv node
-  graph.RemoveInitializedTensor(conv_W_tensor_proto->name());
+  graph_utils::ReplaceInitializer(graph, conv_W_tensor_proto->name(), new_conv_W_tensor_proto);
```

Comment:
Should we be checking this returns true whenever we use it (here and elsewhere)?

Reply:
we could, but the only way it would return false is if the original_name wasn't found, and that shouldn't be possible given we get the name from an initializer we just looked up. may be cleaner to make ReplaceInitializer throw if original_name isn't found.

Reply:
Sure, that works too.